android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -108,6 +107,14 @@ public void setSrc(FastImageViewWithUrl view, @Nullable ReadableMap source) {
         }
     }
 
+    @ReactProp(name = "gradient")
+    public void setGradient(FastImageViewWithUrl view, @Nullable ReadableMap gradient) {
```

Comment:
Blend mode and gradient are two different features, I think we should keep separate properties for them